test/deploy/linux/php/templates/install-wordpress.sh

```diff
@@ -56,12 +56,20 @@ if [ $(which nginx) ]; then
 fi
 
 if [ $(which apache2) ]; then
-	if [ ! -f /etc/apache2/sites-available/wordpress-site ]; then
+	SITE="wordpress"
+	if [[ $(dpkg -l php-fpm) ]]; then
```

Comment:
Is this intentionally Debian only?

The file name to me feels generically Linux, but `dpkg` is strictly deb right? Other than potentially this, the PR looks great though!

Reply:
Good eye! The original intention was for it to be OS agnostic, but `php-fpm` doesn't get install anywhere on the path so running `which php-fpm` wasn't sufficient to determine if it is installed. `dpkg` is absolutely Debian specific. We will either need to do some OS detection here to specialize the command used or find a reliable OS agnostic way to detect whether `php-fpm` is installed.

I'm inclined to leave this as-is and address this once we formally add support for RedHat.